src/oge/data_cleaning.py

```diff
@@ -720,6 +720,50 @@ def clean_cems(year: int, small: bool, primary_fuel_table, subplant_emission_fac
     # create an inventory of which plant-months have input data from any source
     inventory_input_data_sources(cems, year)
 
+    # Flag outliers
```

Comment:
Can we move all of this code into `validation` so that this is just a one-line call to the validation check like the others?

Reply:
Done